docker-compose.yml

```diff
@@ -70,12 +70,15 @@ services:
         ports:
             - "127.0.0.1:5672:5672"
     ddagent:
-        image: datadog/docker-dd-agent
+        image: datadog/agent-dev:gbbr-apm-build
         environment:
             - DD_BIND_HOST=0.0.0.0
             - DD_API_KEY=invalid_key_but_this_is_fine
+            - DD_APM_RECEIVER_SOCKET=/tmp/ddagent/trace.sock
```

Comment:
Does this add an additional binding for the socket, or is it by host/port or UDS only?

Reply:
Answer is here: https://github.com/DataDog/datadog-agent/pull/3556/files#diff-f4f2947f405ab01c53d35418cdb59bebR149

It adds an additional binding.

Reply:
cool, I figured because the tests were still all passing, so 👍